app/src/main/java/cl/mario/covid/routermodule/service/RouterManager.kt

```diff
@@ -0,0 +1,11 @@
+package cl.mario.covid.routermodule.service
+
+import cl.mario.covid.covidmodule.service.CovidHelperRouter
+import cl.mario.covid.covidmodule.service.CovidServiceImpl
+
+object RouterManager {
+
+    var covidServiceImpl = CovidServiceImpl().apply {
+        CovidHelperRouter.covidService = this
+    }
```

Comment:
Everything is fine, but this could be better 🌟 
you need to think, what should happen if you need to integrate another module?
Your router could supported? Could be easy the implementation for other developers?

Maybe your RouterManage need other thing 🤔 
Something Generic? Maybe! general interfaces in the other modules? Could be!
Other Module? (Library with general Utils/clasess/interfaces)